### PR TITLE
HOTFIX: native timeline manager constructor

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
@dephillipsmichael

### CODE REVIEW

File: bridge-client/src/iosMain/kotlin/org/sagebionetworks/bridge/kmm/shared/NativeTimelineManager.kt [hunk 1]
```diff
@@ -18,9 +20,9 @@ import platform.Foundation.*
 
 class NativeTimelineStudyBurstManager(
     private val studyId: String,
-    scheduleMutator: ParticipantScheduleMutator? = null,
+    scheduleMutator: ParticipantScheduleMutator?,
     private val viewUpdated: (NativeStudyBurstSchedule?, String?) -> Unit,
-    private val updateFailed: (() -> Unit)? = null
```
Comment (dephillipsmichael) [inline bridge-client/src/iosMain/kotlin/org/sagebionetworks/bridge/kmm/shared/NativeTimelineManager.kt:23]: Ahhh yeah, as you know the optional parameters get ignored so there's no point in adding them for iOS-only code.

**Comment by dephillipsmichael** [comment]
@syoung-smallwisdom looks good to me, but the new unit test failed.

**Comment by dephillipsmichael** [comment]
The DIAN app couldn't reference version 0.6.6. Do you need to tag branch to trigger a build?

**Comment by syoung-smallwisdom** [comment]
> The DIAN app couldn't reference version 0.6.6. Do you need to tag branch to trigger a build?

Yes

**Review by nategbrown9 — APPROVED** [review]

**Pull request merged**